ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/targetdefinition/TargetEditor.java

```diff
@@ -406,8 +408,19 @@ public void run() {
 		export.setToolTipText("Export"); //$NON-NLS-1$
 		export.setImageDescriptor(PDEPluginImages.DESC_EXPORT_TARGET_TOOL);
 
+		Action openTPPreference = new Action("openTPPreference") { //$NON-NLS-1$
+			@Override
+			public void run() {
+				String pageId = TargetPlatformPreferencePage.PAGE_ID;
```

Comment:
It's a nit, but this assignment seems pointless for a a constant used only once.

Reply:
You're right, copy-pasted that but having that assignment doesn't make sense.